templates/etc/apt/sources.list.d/nodesource.list

```diff
@@ -1,2 +1,2 @@
-deb https://deb.nodesource.com/node_10.x buster main
-deb-src https://deb.nodesource.com/node_10.x buster main
+deb https://deb.nodesource.com/node_12.x buster main
+deb-src https://deb.nodesource.com/node_12.x buster main
```

Comment:
Good to get this upgraded! Thanks for remembering that the old uploader was the blocker on this, and upgrading at the same time as removing that blocker.

For anyone else following along: the 10.x line is in maintenance LTS status, and leaves maintenance on 2021-04-30; the 12.x line is in active LTS status, and leaves maintenance on 2022-04-30. See [Node.js § Releases](https://en.wikipedia.org/wiki/Node.js#Releases). Next year we should see about upgrading to 14, but we haven't tested that on either the new upload service or on the frontend, so that's way out of scope for now.